Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs

```diff
@@ -67,7 +67,7 @@ public static async Task<GltfObject> ImportGltfObjectFromPathAsync(string uri)
                     return null;
                 }
             }
-            else if (uri.EndsWith(".glb"))
+            else if (uri.ToLower().EndsWith(".glb"))
```

Comment:
```suggestion
            else if (uri.EndsWith(".glb", StringComparison.OrdinalIgnoreCase))
```

Reply:
of course way better! thx for pointing this out and done in new commit